packages/web-components/examples/codesandbox/components/card-group/cdn.html

```diff
@@ -52,7 +52,7 @@
               <path
                 d="M11.8 2.8L10.8 3.8 16.2 9.3 1 9.3 1 10.7 16.2 10.7 10.8 16.2 11.8 17.2 19 10z"></path>
             </svg>
-          </dds-card-footer>
+          </dds-card-cta-footer>
```

Comment:
@ariellalgilmore is this an API change or a correction in the example?

Reply:
correction in the example. card-group should always use `<dds-card-cta-footer>`. `<dds-card-group-item>` extends `DDSCardCTA` so the footer should match.

Reply:
Ok, we might want to note it in the next release to change to `dds-card-cta-footer` if this change breaks for anyone using `dds-card-footer` instead.

Reply:
yeah good idea!